newsfragments/2412.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+* Remove leftover imports of Twisted Logger, using instead our shim.
+* Change NuCypher Logger behavior to always escape curly braces. The rationale is that it's unclear how to justify the previous behavior with some edge cases like ``{{}``, which was processed to ``{}``.
```

Comment:
current text produces weird sub-bullets:

<img width="733" alt="Screen Shot 2020-11-02 at 10 26 04 AM" src="https://user-images.githubusercontent.com/19150641/97886989-4f6d6000-1cf7-11eb-8c97-b26c39a71f30.png">


You can use the line block syntax if you like - https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#line-blocks

```suggestion
| Remove leftover imports of Twisted Logger, using instead our shim.
| Change NuCypher Logger behavior to always escape curly braces. The rationale is that it's unclear how to justify the previous behavior with some edge cases like ``{{}``, which was processed to ``{}``.
```

which produces:

<img width="771" alt="Screen Shot 2020-11-02 at 10 31 18 AM" src="https://user-images.githubusercontent.com/19150641/97887070-62803000-1cf7-11eb-817c-13b02e2c10f9.png">